src/dbscan.jl

```diff
@@ -7,23 +7,53 @@
 #       in large spatial databases with noise. 1996.
 #
 
+"""
+The output of [`dbscan`](@ref) function (distance matrix-based implementation).

```

Comment:
I think the convention is to put this on the first line, followed by a line break:
```
    DbScanResult
```